rbac/providers/ldap/ldap_message_validator.py

```diff
@@ -19,19 +19,17 @@ def validate_next_payload(payload):
     """Confirms the payload has the fields required for ldap message transformation"""
 
     # TODO: Move these into an enum, share with outbound_sync
+
     required_field_data = "data"
     required_field_data_type = "data_type"
     required_field_provider_id = "provider_id"
     required_value_data_type = ["user", "group"]
     required_field_dn = "distinguished_name"
-
-    # TODO: These are also required given the current outbound_sync mappings. Include tests!
-    #       Enforce or remove from outbound_sync
-    # required_field_user_name = "data.user_name"
-    # required_field_cn = "data.cn"
-    # required_field_given_name = "data.given_name"
-    # required_field_name = "data.name"
-    # required_field_manager = "data.manager"
+    required_field_user_name = "data.user_name"
+    required_field_cn = "data.cn"
+    required_field_given_name = "data.given_name"
+    required_field_name = "data.name"
+    required_field_manager = "data.manager"
```

Comment:
These changes look good. Unfortunately, the TODO is a bit out of date since the mappings have been removed from outbound sync. 

There is a second part of this process where the values are mapped before going to AD, and the mappings will need to happen in order to include the fields in the message and prove out the integration on both ends (Next and Ldap). 

Before this change goes in, it makes sense to map these fields first and confirm their consistency with AD. If it rejects and/or doesn't map them, then the changes here will need to be revisited individually. 

It was a mistake on my part not to overview this with you and in assuming validation could be worked on separately from mapping. They both need to happen together. That said: The work you have done here will need to be incorporated into the larger feature of the sync, and I hope you are up to taking in on because the value-added moves up from test coverage to feature implementation (ie it will be an interesting exercise that will add a lot of value). 

Marking this as "needs work". Not because of code issues, but because the feature itself needs it. We can sort this out either by merging this and rectifying shortly thereafter or by adding the second part to this story. Discussion soon!